sintia/sintia.py

```diff
@@ -821,6 +830,29 @@ async def get_supported_symbols() -> set[str]:
                 f'{amount:.2f} {from_unit} = {converted:.2f} {to_unit}',
             )
 
+    @command_handler('chat')
+    async def chat(self, message: GenMessage, argument: str) -> None:
+        return await message.channel.send(self.chat_brain.make_sentence())
+
+    @command_handler('savebrain')
+    async def save_brain(self, message: GenMessage, argument: str) -> None:
+        with open('brain.json', 'wt') as f:
+            f.write(self.chat_brain.to_json())
+
+        return await message.add_reaction('✔')
+
+    async def chat_message_handler(self, message: discord.Message) -> None:
```

Comment:
```suggestion
    async def chat_message_handler(self, irc_bridge: IrcBridge, message: discord.Message) -> None:
```

Reply:
but why? the function is already bound to `self`

Reply:
oh, yeah, I forgot that this was not part of the `command_handler`, indeed, this can use the `self.irc_bridge`, and just use the oneliner below to fix this